astropy/coordinates/tests/test_pickle.py

```diff
@@ -6,9 +6,9 @@
 import astropy.units as u
 from astropy import coordinates as coord
 from astropy.coordinates import Longitude, representation
-from astropy.tests.helper import check_pickling_recovery, pickle_protocol  # noqa
+from astropy.tests.helper import check_pickling_recovery, pickle_protocol  # noqa: F401
```

Comment:
Is this an unused import? Or is it because ``flake8`` doesn't understand that this is being used as a fixture?

Reply:
This is one of those unfortunate cases of `pytest` not being very "pythonic", which means, as you suggested, `flake8` doesn't understand its being used.